build/mediaelement.js

```diff
@@ -2720,6 +2720,7 @@ var FlashMediaElementRenderer = {
 			};
 		};
 		methods.push('stop');
+        methods.push('canplay');
```

Comment:
Why do you need a `canplay` method, if the `canplay` is actually an event, and it is passed later in the code?